src/os/bluestore/BitAllocator.cc

```diff
@@ -355,8 +355,7 @@ void BitMapZone::init(int64_t zone_num, int64_t total_blocks, bool def)
   alloc_assert(total_blocks < std::numeric_limits<int32_t>::max());
   alloc_assert(!(total_blocks % BmapEntry::size()));
 
-  BmapEntryVector *bmaps = new BmapEntryVector(num_bmaps, BmapEntry(cct, def));
-  m_bmap_list = bmaps;
+  m_bmap_vec = BmapEntryVector(num_bmaps, BmapEntry(cct, def));
```

Comment:
You may want to use resize() method instead that's probably a bit more effective.